flyteidl/protos/flyteidl/core/errors.proto

```diff
@@ -25,6 +25,12 @@ message ContainerError {
 
     // Defines the origin of the error (system, user, unknown).
     ExecutionError.ErrorKind origin = 4;
+
+    // Timestamp of the error
+    int64 timestamp = 5;
+
+    // Worker that generated the error
+    string worker = 6;
```

Comment:
Haven't looking into the rest of the PR yet. I assume this is referring to a distributed pytorch plugin worker in case of pytorch plugin?

Reply:
The RFC made this a generic field. In the case of PyTorch plugin, it is one of the PyTorch worker pods, yes. But for other plugins, it could mean something else.